### ISSUE TITLE
React.Strict breaks the inspector

### ISSUE THREAD

**Issue by diamond-darrell** [statelyai/inspect#44]
Hello there, 
I've noticed an issue with the inspector when React.Strict mode is enabled.
Basically, it gets stuck on actor execution after rerendering the component that provided the machine. The machine's runtime execution seems fine since we're reaching the idle state, but it makes the inspector unusable afterward. You can see it's working if the strict mode is disabled.

### CodeSandbox
https://codesandbox.io/p/sandbox/stately-inspect-iframe-repro-forked-hwv7ch

**Comment by davidkpiano** [comment]
One small note: `createBrowserInspector()` is meant to be called _outside_ of components; that may be a potential source of the problem. Can you try something like this instead?

```ts
const inspector = createBrowserInspector();

function Component() {
  // the rest of your code
}
```

**Comment by diamond-darrell** [comment]
@davidkpiano yes, I've updated the codesandbox example. The issue still happens

**Comment by tobias-spv** [comment]
I'm experiencing the same issue 🤔
